pyvista/plotting/plotter.py

```diff
@@ -4218,17 +4302,17 @@ def add_volume(
         assert_empty_kwargs(**kwargs)
 
         if show_scalar_bar is None:
-            show_scalar_bar = self._theme.show_scalar_bar or scalar_bar_args
+            show_scalar_bar = self._theme.show_scalar_bar or scalar_bar_args  # type: ignore[assignment]
 
         # Avoid mutating input
         scalar_bar_args = {} if scalar_bar_args is None else scalar_bar_args.copy()
 
         if culling is True:
-            culling = 'backface'
+            culling = 'backface'  # type: ignore[assignment]
```

Comment:
Ignore can probably be removed once `backface` is added to the hinted `Literal`
```suggestion
            culling = 'backface'
```